web/docs/tql2/operators.md

```diff
@@ -101,14 +105,11 @@ Operator | Description | Example
 [`save_udp`](./operators/save_udp.md) | Saves incoming bytes to a UDP socket | `save_udp "0.0.0.0:8090"`
 [`save_zmq`](./operators/save_zmq.md) | Saves incoming bytes to ZeroMQ messages | `save_zmq`
 [`serve`](./operators/serve.md) | Makes events available at `/serve` | `serve "abcde12345"`
-[`to_azure_log_ana…`](./operators/to_azure_log_analytics.md) | Sends events to Azure Log Analytics | `to_azure_log_analytics tenant_id=…`
+[`to_azure_log_analytics`](./operators/to_azure_log_analytics.md) | Sends events to Azure Log Analytics | `to_azure_log_analytics tenant_id=…`
+[`to_fluent_bit`](./operators/to_fluent_bit.md) | Sends events to Fluent-Bit| `to_fluent_bit fluent-bit "es" options= {host=…`
```

Comment:
Capitalization of "Fluent-Bit" is inconsistent with the description of `from_fluent_bit`. 

The example is broken.